pkg/services/app/add.go

```diff
@@ -120,6 +120,13 @@ func (a *App) Add(params AddParams) error {
 		return fmt.Errorf("could not update parameters: %w", err)
 	}
 
+	if params.SourceType != string(wego.SourceTypeHelm) {
```

Comment:
It seems there is no logic reason right now. However a probable explanation is that we follow this patter on fields that are used by cobra to read parameters because cobra can't read them using custom data types, or at least not in the way we are reading them with cobra right now. I'll fix that.